src/_docs/services/hierarchical.data.ts

```diff
@@ -4589,6 +4589,81 @@ export let SampleHierarchicalData = [
     {'id': '4586', 'parentId': '4584', 'name': 'Level 7', 'subtext': 'id 4586', 'type': 'Level 7'}
 
```

Comment:
@thomastighe Can you ensure this isn't part of the startup bundle? It needs to be either lazy loaded with the module for your example, or make an http call to fetch the static file from the server. 

No issues here as long as that's the case. We just don't want our initial load and bundle having static data that may or may not need to be downloaded by our users.